scripts/jenkins/ardana/ansible/roles/ardana_update/tasks/update_nodes.yml

```diff
@@ -0,0 +1,83 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Refresh repo(s) on all ardana nodes
+  command: |
+    ansible -b -a "zypper --non-interactive --gpg-auto-import-keys refresh"
+      'resources:!*rhel*'
+  args:
+    chdir: "{{ ardana_scratch_path }}"
+
+- name: Run ardana-update-pkgs playbook for deployer
+  shell: |
+    ansible-playbook ardana-update-pkgs.yml \
+      -l ARD-SVC--first-member -e \
+      "zypper_update_method={{ update_method }}
```

Comment:
We might want to use `update` for some cases, and `patch` for other, depending on what we're updating. The `patch` method is what we recommend customers to do, to apply maintenance updates, while `update` is what we have to do internally in CI most of the time, when we don't test maintenance updates, or when maintenance updates don't include the "patch" meta-packages. So the default should be `update`, but in a future PR we might want to make this a job parameter.

Reply:
The current value is, if updating to a devel/staging cloudsource use `update` else use `patch`. So for maintenance updates we will continue using `patch`.

Reply:
excellent !